handbook/HR/expenses-policy.md

```diff
@@ -0,0 +1,27 @@
+## Expenses Policy
+
+#### Reimbursement of expenses
```

Comment:
Couple of points:

1) We don't currently have an expense review processes
2) I think we should probably set out some basic frameworks for people to make decisions about:

- total monthly limit
- max cost of travel; intra-UK, intra-EU, transatlantic, international

Be glad to hear your thoughts?